File: src/main/java/com/teno/Hello.java
package com.teno;
public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello World");
//        new Person().hello();  //new Person物件出來

        Person p = new Person();  //等號右邊就是先在記憶體當中生出一個(Person)物件,在將物件的地址寫在一張叫p的紙上面
        p.hello();  //用p去呼叫hello這個方法
    }
}

File: src/main/kotlin/com/teno/Hello.kt
package com.teno

fun main() {
//    println("Hello kotlin")
//    Human().hello()  //在kotlon裡面,呼叫建構子不需要new

    //儲存物件
    val h = Human()  //val(Value)不能改變它的值
    h.hello()
}

class Human{
    fun hello(){
        println("Hello World")
    }
}
